environment-dev.yml

```diff
@@ -1,12 +1,12 @@
 name: mantidimaging-dev
 channels:
-  - mantidimaging/label/unstable
+  - mantidimaging/label/main
   - conda-forge
   - ccpi
   - algotom
   - https://software.repos.intel.com/python/conda/
 dependencies:
-  - mantidimaging>=2.5.0
+  - mantidimaging>=2.8.0,<3.1.0
```

Comment:
I think this (and the other one) should be `mantidimaging>=3.0.0,<3.1.0` so it forces a 3.0 install.

See https://github.com/mantidproject/mantidimaging/blob/release-2.8.0/environment.yml